automation_tools/__init__.py

```diff
@@ -721,6 +721,13 @@ def downstream_install(admin_password=None, run_katello_installer=True):
         remote_path='/etc/yum.repos.d/satellite.repo')
     satellite_repo.close()
 
+    info = distro_info()
+    if info == ('rhel', 6, 6):
+        # For default java-1.8.0-openjdk will be installed on RHEL 6.6 but it
+        # makes the katello-installer fail. Install java-1.7.0-openjdk which is
+        # the recommended version for RHEL 6.6.
+        run('yum install -y java-1.7.0-openjdk')
+
```

Comment:
I did the following workaround to get installation in rhel 6.6 going, any reason you are using only one of the three steps which I mentioned below?

```sh
yum -y install java-1.7.0-openjdk
alternatives --config java 
--> (selected 1.7.0 here)
# rpm -e java-1.8.0-openjdk-headless-1.8.0.45-28.b13.el6_6.x86_64 java-1.8.0-openjdk-1.8.0.45-28.b13.el6_6.x86_64
```